src/makeRobot.js

```diff
@@ -38,7 +38,68 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      'x': 0,
+      'y': 0,
```

Comment:
no need to specify that it's a string
```suggestion
      x: 0,
      y: 0,
```